meson.build

```diff
@@ -333,6 +333,7 @@ possible_cc_flags = [
         '-Wno-missing-field-initializers',
         '-Wno-unused-result',
         '-Wno-format-signedness',
+        '-Wno-#warnings',
```

Comment:
hmm, what does this precisely do? turn off the whole effect of `#warning` (i.e. is the message specified now even shown either) or just that it is considered no relevant warning anymore, in particular for the purpose of -Werror (i.e. the msg is still shown)?

Reply:
Yeah, not sure. I hoped that this would just not make warnings errors. But it seems it does more.